BlueprintUI/Sources/Layout/SPL/SPLayout.swift

```diff
@@ -6,14 +6,26 @@ public protocol SPLayout {
 
     typealias Subviews = LayoutSubviews
 
+    associatedtype Cache = Void
+
     func sizeThatFits(
         proposal: SizeConstraint,
-        subviews: Subviews
+        subviews: Subviews,
+        cache: inout Self.Cache
```

Comment:
is the `Self.` here intentional/needed?

Reply:
Hm, doesn't seem needed, but I don't recall.